shardingsphere-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,206 @@
+registryItems:
+  applications:
+    - Your_ApplicationName: nq 0
+  instances:
+    - Your_ApplicationName: 1
+  operationNames:
+    - Your_ApplicationName:
+        - "/case/execute"
+segmentItems:
+  - applicationCode: Your_ApplicationName
+    segmentSize: not null
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: "/ShardingSphere/parseSQL/"
+            operationId: '0'
+            parentSpanId: "-1"
+            spanId: "0"
+            spanLayer: Unknown
+            tags:
+              - key: db.statement
+                value: CREATE TABLE IF NOT EXISTS t_order (order_id BIGINT NOT NULL AUTO_INCREMENT,
+                  user_id INT NOT NULL, status VARCHAR(50), PRIMARY KEY (order_id))
+            startTime: not null
+            endTime: not null
+            componentId: '60'
+            componentName: ''
+            isError: false
+            spanType: Local
+            peer: ''
+            peerId: '0'
+      - segmentId: not null
+        spans:
+          - operationName: "/ShardingSphere/executeSQL/"
+            operationId: '0'
+            parentSpanId: "-1"
+            spanId: "0"
+            spanLayer: Unknown
+            refs:
+              - parentEndpointId: 0
+                parentEndpoint: "/ShardingSphere/JDBCRootInvoke/"
+                networkAddressId: 0
+                entryEndpointId: 0
+                refType: CrossThread
+                parentSpanId: 0
+                parentTraceSegmentId: not null
```

Comment:
I suggest that you should make sense the value of `parentTraceSegmentId`

Reply:
The parent trace segmentId is unpredictable. During the last test is segmentId is .1.15562751912310010. Any suggestion?